src/main/java/net/dries007/tfc/util/Helpers.java

```diff
@@ -557,6 +571,57 @@ public static boolean spawnItem(World world, BlockPos pos, ItemStack stack)
         return perfectMatchDet(matrices, size);
     }
 
+    public static Triple<Integer, Float, Long> consumeFuelForTicks(long deltaPlayerTicks, IItemHandlerModifiable inventory, int burnTicks, float burnTemperature, int slotStart, int slotEnd)
```

Comment:
This needs at least a doc comment saying what it does (i.e. what the spec is) and what on earth it is returning. (god how I wish this could return a record :pensive: